src/handler.ts

```diff
@@ -98,16 +100,29 @@ export async function handlePullRequest(
   }
 
   if (addReviewers) {
-    try {
-      const reviewers = utils.chooseReviewers(owner, config)
-
-      if (reviewers.length > 0) {
-        await pr.addReviewers(reviewers)
-        core.info(`Added reviewers to PR #${number}: ${reviewers.join(', ')}`)
+    if (teamReviewers && teamReviewers.length > 0) {
+      try {
+        await pr.addTeamReviewers(teamReviewers)
+        core.info(
+          `Added team_reviewers to PR #${number}: ${teamReviewers.join(', ')}`
+        )
+      } catch (error) {
+        if (error instanceof Error) {
+          core.warning(error.message)
+        }
       }
-    } catch (error) {
-      if (error instanceof Error) {
-        core.warning(error.message)
+    } else {
```

Comment:
I don't think reviewers and teamReviewers should be mutually exclusive. Is there any limitation to have both a user and a team to review on a PR?